app/frontend/components/domains/permit-application/new-permit-application-screen.tsx

```diff
@@ -192,9 +195,12 @@ export const PinModeInputs = () => {
           render={({ field: { onChange, value } }) => {
             return (
               <JurisdictionSelect
-                onChange={onChange}
+                onChange={(value) => {
+                  addJurisdiction(value)
+                  onChange(value)
```

Comment:
is the value here a Jurisdiction model or an `IOption`

Reply:
its a bit confusing and it took some intense study, but the value is a Jurisdiction but the selectedOption is an IOption